.ci/build-packages.sh

```diff
@@ -358,7 +332,110 @@ else
+
+# Now that we have published the new version we need to create
+# a new branch for the versioned docs and push the gem as a release
+# into the vagrant repository.
+tags=($(git tag -l --sort=v:refname)) # this provides tag list in ascending order
+tag_len="${#tags[@]}"
+for ((i=0; i < "${tag_len}"; i++))
+do
+    val=$(semver "${vagrant_version}" "${tag}")
+    if [[ "${val}" == "0" ]]; then
+        idx=$((i+1))
+        break
+    fi
+done
+if [ -z "${idx}" ]; then
+    fail "Failed to determine previous version from current release - ${vagrant_version}"
+fi
+previous_version="${tags[$idx]}"
+if [ -z "${previous_version}" ]; then
+    fail "Previous version detection did not provide valid result. (Current release: ${vagrant_version})"
 fi
+previous_version="${previous_version:1:${#previous_version}}" # remove 'v' prefix
+
+# Now that we have the previous version, checkout the stable-website
+# branch and create a new release branch.
+release_branch="release/${previous_version}"
+release_minor_branch="release/${previous_version%.*}.x"
+echo "Creating a new release-${previous_version} branch..."
+
+wrap git checkout stable-website \
+     "Failed to checkout stable-website branch"
+wrap git checkout -b "${release_branch}" \
+     "Failed to create new branch: ${release_branch}"
+wrap git push origin "${release_branch}" \
+     "Failed to push new branch: ${release_branch}"
+
+echo "Creating a new ${release_minor_branch} branch..."
+wrap git checkout stable-website \
+     "Failed to checkout stable-website branch"
+git branch -d "${release_minor_branch}" > "${output}"
+
+wrap git checkout -b "${release_minor_branch}" \
+     "Failed to create new branch: ${release_minor_branch}"
+wrap git push -f origin "${release_minor_branch}" \
+     "Failed to push new branch: ${release_minor_branch}"
+
+slack -m "New branches created for previous release: ${release_branch} and ${release_minor_branch}"
+
+# Now update the stable-website branch with new version content
+echo "Updating stable-website with latest..."
+wrap git checkout "v${vagrant_version}" \
+     "Failed to checkout vagrant release v${vagrant_version}"
+wrap git branch -d stable-website \
+     "Failed to delete stable-website branch"
+wrap git checkout -b stable-website \
+     "Failed to create new stable-website branch from release v${vagrant_version}"
+wrap git push -f origin stable-website \
+     "Failed to push updated stable-website branch"
+
+slack -m "Vagrant website changes have been pushed for release ${vagrant_version}"
+
+# Finally, we need to create a release on the vagrant repository
+wrap mv ../"vagrant-${vagrant_version}.gem" ./
+
+# Override local variables to create release on vagrant repository
+repo_owner="hashicorp"
+repo_name="vagrant"
+full_sha="v${vagrant_version}"
+export GITHUB_TOKEN="${HASHIBOT_TOKEN}"
+
+release "v${vagrant_version}" ./"vagrant-${vagrant_version}.gem"
```

Comment:
@chrisroberts what binary does `release` refer to?

Will this create a GitHub **release** in the Vagrant repo?

Reply:
yes, the `repo_name` is set to the repository the workflow is running on (in this case it will be `vagrant-builders`) and just above on line 435 we update the name to `vagrant` so the release will be created on the `hashicorp/vagrant` repository